logstash-core.gemspec

```diff
@@ -0,0 +1,80 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path('../lib/logstash/version', __FILE__)
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["Jordan Sissel", "Pete Fritchman"]
+  gem.email         = ["jls@semicomplete.com", "petef@databits.net"]
+  gem.description   = %q{The core components of logstash, the scalable log and event management tool}
+  gem.summary       = %q{logstash-core - The core components of logstash}
+  gem.homepage      = "http://logstash.net/"
+  gem.license       = "Apache License (2.0)"
+
+  gem.files         = `git ls-files`.split($\).select do |file| /^lib\/|^spec\//.match(file) end
```

Comment:
Ok! Can do glob ;-) I just wanted to keep the style with the old one, but Dir.glob looks even better here.